### PR TITLE
Add Web Rebels and Trondheim Developer Conference

### PR DESCRIPTION
**Pull request by Krakels**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +88,7 @@ Additions/corrections are welcome! Please check out [Contributing Guidelines](CO
 | [CSSConf EU](http://cssconf.eu) | Berlin, Germany | June 1, 2018 | January 1, 2018 | Yes | [Yes](https://2017.cssconf.eu/code-of-conduct/) |
 | [Amsterdam JS](http://amsterdamjs.com) | Amsterdam, The Netherlands | June 1, 2018 | March 1, 2018 | Yes | [Yes](https://http://confcodeofconduct.com/) |
 | [JSConf EU](http://jsconf.eu) | Berlin, Germany | June 2 - 3, 2018 | January 1, 2018 | Yes | [Yes](http://2017.jsconf.eu/code-of-conduct) |
+| [Web Rebels](https://webrebels.org/) | Oslo, Norway | June 4 - 5, 2018 | February 25, 2018 | Unknown | Yes |
```
Comment (benmvp) [inline README.md]: Looks like Web Rebels does indeed cover travel & accommodations: https://www.papercall.io/cfps/848/submissions/new. Can you update to say `Yes`? Thanks!

**Comment by benmvp** [comment]
Thanks for your additions!

File: README.md (hunk 1 above)
Comment (Krakels) [reply README.md]: Good catch! This has now been fixed.

**Review by benmvp — APPROVED** [review]
Thanks again!

**Pull request merged**